sw/device/lib/testing/mock_csr_test.cc

```diff
@@ -0,0 +1,42 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/mock_csr.h"
+
+#include "gtest/gtest.h"
+#include "sw/device/lib/base/csr.h"
+
+namespace {
```

Comment:
Missing outer namespace. =)

Reply:
I don't _think_ we need one here? This is just an anonymous namespace for the tests. `mock_mmio_test.cc` uses the same pattern.

Reply:
I've put them into a namespace (`mock_csr_test`) for now.